src/execution/execution_entry_point.rs

```diff
@@ -578,4 +628,170 @@ impl ExecutionEntryPoint {
             call_result,
         )
     }
+
+    #[cfg(not(feature = "cairo-native"))]
```

Comment:
Let's add `#[inline(always)]` in this case to help the dead code removal figure out the `match` always end up in the same branch.

Reply:
Fixed.